ecl/hql/reservedwords.cpp

```diff
@@ -576,3 +602,65 @@ void printKeywordsToXml()
      buffer.append("</xml>");
      fprintf(stdout, "%s\n", buffer.str());
 }
+
+static const char * internalTokens[] = { //tokens present in the grammar that aren't in the language or are waiting to be deprecated
+    "",
+    "#elif",
+    "&&",
+    "constant",
+    "complex-macro",
+    "c++", //"c++" is in the actual above lists however, getTokenText() returns without " c.f. HPCC-15021
```

Comment:
This isn't an internal token, in that it can occur in the syntax

     EMBED(C++)

It is correct to return without surrounding quotes